docs/changes/visualization/17006.feature.rst

```diff
@@ -0,0 +1,2 @@
+Allow :obj:`astropy.visualization.units.quantity_support` to be used as either
```

Comment:
I believe RTD is failing because of this ref. We usually avoid true references in the changelog so it doesn't rot in the future if we ever removed referenced items.

```suggestion
Allow ``astropy.visualization.units.quantity_support`` to be used as either
```

Reply:
It is a bit weird, though, that it isn't found. It may be because if would be `astropy.visualization.quantity_support`, i.e., one should link to the place one expects users to import from. Indeed, https://docs.astropy.org/en/latest/api/astropy.visualization.quantity_support.html exists. Anyway, agreed that for the changelog entry one should use the double backticks.